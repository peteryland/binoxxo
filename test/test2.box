3x2x1o
1o6o
2xx1x3x
0
2o2oo
o2x
5x1o
xx3x1x
4x4o
ox
