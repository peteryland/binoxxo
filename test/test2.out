___X__X_O_
_O______O_
__XX_X___X
__________
__O__OO___
O__X______
_____X_O__
XX___X_X__
____X____O
OX________

XXOXOOXXOO
XOXOXOXXOO
OOXXOXOOXX
OXOXOXXOOX
XXOOXOOXXO
OOXXOOXXOX
OOXOXXOOXX
XXOXOXOXOO
XOXOXOXOXO
OXOOXXOOXX
